LWA_EPIC/LWA_EPIC.py

```diff
@@ -978,8 +982,8 @@ def main(self):
 
                 oshape = (1, nchan, npol ** 2, self.grid_size, self.grid_size)
                 ogulp_size = nchan * npol ** 2 * self.grid_size * self.grid_size * 8
-                self.iring.resize(igulp_size)
-                self.oring.resize(ogulp_size, buffer_factor=5)
+                self.iring.resize(igulp_size, buffer_factor=128)
+                self.oring.resize(ogulp_size, buffer_factor=256)
```

Comment:
Here is another large buffer in `MOFFCorrelatorOp`.  The `buffer_factor` of 256 on the output ring
could be one of the big things driving the GPU memory utilization.